src/block/utils/AnvilHelper.php

```diff
@@ -0,0 +1,189 @@
+<?php
+
+/*
+ *
+ *  ____            _        _   __  __ _                  __  __ ____
+ * |  _ \ ___   ___| | _____| |_|  \/  (_)_ __   ___      |  \/  |  _ \
+ * | |_) / _ \ / __| |/ / _ \ __| |\/| | | '_ \ / _ \_____| |\/| | |_) |
+ * |  __/ (_) | (__|   <  __/ |_| |  | | | | | |  __/_____| |  | |  __/
+ * |_|   \___/ \___|_|\_\___|\__|_|  |_|_|_| |_|\___|     |_|  |_|_|
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * @author PocketMine Team
+ * @link http://www.pocketmine.net/
+ *
+ *
+ */
+
+declare(strict_types=1);
+
+namespace pocketmine\block\utils;
+
+use pocketmine\inventory\transaction\TransactionValidationException;
+use pocketmine\item\Durable;
+use pocketmine\item\EnchantedBook;
+use pocketmine\item\enchantment\AvailableEnchantmentRegistry;
+use pocketmine\item\enchantment\Enchantment;
+use pocketmine\item\enchantment\EnchantmentInstance;
+use pocketmine\item\enchantment\Rarity;
+use pocketmine\item\Item;
+use pocketmine\player\Player;
+use function ceil;
+use function floor;
+use function max;
+use function min;
+use function strlen;
+
+class AnvilHelper{
```

Comment:
I would like this class to be divided into actions, for example an AnvilRapairAction, in the actions would define properties such as the cost and process the item

Reply:
Like the idea